e2e_tests/tests/fixtures/pytorch-no-op/model_def.py

```diff
@@ -1,3 +1,4 @@
+import logging
```

Comment:
I figure leave this as is, move it and just add the logging for now. If we need to start toggling behaviors off and on like in the no op tests, we can do it when we need to. I didn't add that to an existing parallel test because they all use stuff under examples/ and I don't want to add garbage random logs.